python/lsst/ip/isr/calibType.py

```diff
@@ -555,10 +630,8 @@ def fromTable(cls, tableList):
         table = tableList[0]
         metadata = table.meta
         inDict = dict()
+
```

Comment:
Some people care deeply about random whitespace additions, but I am not one of those people 🙂